src/commandLine/CommandLine.ts

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env node
+/* eslint-disable dot-notation */
+
+import * as pp from './printPretty';
+import * as help from './helpers';
+import { getArgs } from './args';
+const log = console.log;
+const RR = require('../modules/RokulRunnings').RokulRunnings;
```

Comment:
why the mixing of `require` and `import` here?

Reply:
Was having issues with getting them working and never circled back around. I've fixed that now.

Reply:
Fixed in 9262563
FWIW - Configstore was behaving _weird_ without the require so I left it as require